<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Divya" failures="1" tests="2" name="com.testautomation.tests.apiTestSuiteRunner" time="1.793" errors="0" timestamp="2025-03-24T11:31:58 GMT" skipped="0">
  <testcase classname="com.testautomation.tests.apiTestSuiteRunner" name="runScenario" time="0.215">
    <failure message="expected [201] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [201] but found [400]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.testautomation.stepdefinitions.APIStepdefinition.theResponseStatusShouldBe(APIStepdefinition.java:54)
at âœ½.the response status should be 201(file:///C:/Users/vicho/eclipse-workspace/CucumberMavenProject/src/test/resources/features/API/APITest.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.testautomation.tests.apiTestSuiteRunner" name="runScenario" time="1.578"/>
  <system-out/>
</testsuite> <!-- com.testautomation.tests.apiTestSuiteRunner -->
